### ISSUE TITLE
New Issue details All Events page

### ISSUE THREAD

**Issue by scttcper** [getsentry/sentry#73759]
![image](https://github.com/getsentry/sentry/assets/1400464/bca78eec-fe28-40c1-a69d-02a0531b27df)

The all events link that is a part of the new navigation should take you to a slightly redesigned all events page.

depends on: Redesigned event navigation w/ all events button, Search

**Acceptance Criteria:**

- The “all events” tab is hidden. It might be worth putting this part in its own flag
- Add a date picker to the all events page that filters the all events query.
    - The date picker should persist choices from the issues stream. Ideally this is the same component.

blocked by https://github.com/getsentry/sentry/issues/73758
